kafka-client/src/main/java/dev/responsive/kafka/clients/ResponsiveConsumer.java

```diff
@@ -22,69 +22,56 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.Optional;
 import java.util.regex.Pattern;
 import org.apache.kafka.clients.consumer.Consumer;
 import org.apache.kafka.clients.consumer.ConsumerRebalanceListener;
 import org.apache.kafka.clients.consumer.OffsetAndMetadata;
 import org.apache.kafka.clients.consumer.OffsetCommitCallback;
 import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.utils.LogContext;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class ResponsiveConsumer<K, V> extends DelegatingConsumer<K, V> {
   private final List<Listener> listeners;
-  private final Logger logger;
+  private final Logger log;
 
   private static class RebalanceListener implements ConsumerRebalanceListener {
-    private final Optional<ConsumerRebalanceListener> wrappedRebalanceListener;
+    private final ConsumerRebalanceListener wrappedRebalanceListener;
```

Comment:
Kafka Streams should always pass in a rebalance listener when the main consumer subscribes to topics